src/components/Header/Header.tsx

```diff
@@ -24,28 +26,25 @@ export default function Header({ handleToggleButton }: HeaderProps) {
           boxShadow: "none",
           zIndex: (theme) => theme.zIndex.drawer + 1,
           backgroundColor: ({ palette }) => palette.common.white,
+          paddingY: mobile ? theme.spacing(2) : theme.spacing(1),
+          paddingX: theme.spacing(1),
         }}
       >
         <Toolbar sx={{ position: "relative" }}>
           <IconButton
             size="large"
             edge="start"
-            color="primary"
             aria-label="menu"
             sx={{ mr: 0.5 }}
             onClick={handleToggleButton}
           >
             <MenuIcon />
           </IconButton>
-          <Typography color="primary.light" variant="h5" className={classes.pageTitle}>
+          <Typography color="primary" variant={mobile ? "h5" : "h4"} className={classes.pageTitle}>
             Forkfacts
           </Typography>
           <Box className={classes.rightContent}>
-            <Button
-              color="primary"
-              variant="contained"
-              className={classnames(flexStyles.pageFlexRowContainer, classes.authBtn)}
-            >
+            <Button color="primary" variant="contained" className={classnames(classes.authBtn)}>
```

Comment:
The Sign in Button is too big and does not reflect the [Figma Design](https://www.figma.com/file/JexGuMEfOIWqn6KX8SBy5C/Forkfacts-with-material-theme-(Copy-Colours)?node-id=3155%3A114494&t=JRVKXPqXaYJNtTLd-4)

Reply:
I have addressed this issue in the comparing tab ticket.

Reply:
If I address in this task it can cause merge conflict